chaingpt-research.md

```diff
@@ -41,17 +39,95 @@ Consider using Ubiquity OS as a launchpad of plugins and other products for deve
 - Alex reviews user feedback on ChainGPT.
 This may provide new ideas for leveraging ChainGPT's experience.
 
-**Key actions that can be relevant to us:**
+### Key actions that can be relevant to us
+
+**1. Establish cooperation with an effective crypto marketing agency**
+
+It is known from open sources that ChainGPT used several marketing agencies.
+In reality, there were probably more agencies than are mentioned in open sources.
+
+- Lead marketing agency - Coinband
+
+https://coinband.io/case/chaingpt  
+
+- Website design - Sigma Software 
+
+https://sigma.software/about/media/a-winning-streak-for-our-design-teams-new-project-chaingpt
+
+https://dribbble.com/shots/23245957-ChainGPT-Website-Behance-Case-Study  
+
+https://www.behance.net/gallery/186506369/ChainGPT-Blockchain-AI-Website 
+
+- One of the additional marketing agencies - EmoneyMax
+
+https://emoneymax.net/  
+
+- ChainGPT itself provides marketing services in the scope of the ChainGPT Pad 
+
+https://medium.com/@chaingpt/chaingpt-pad-2023-a-year-of-unprecedented-success-in-web3-11e7db93615c 
```

Comment:
Interesting but I'm a bit skeptical. I need to spend more time reading through these carefully. Are there any salient points you want to point out for me?